orderbook/src/api/filter.rs

```diff
@@ -235,20 +262,21 @@ pub mod test_util {
     }
 
     #[tokio::test]
-    async fn get_fee_info_() {
-        let filter = get_fee_info();
-        let sell_token = String::from("0x000000000000000000000000000000000000000a");
-        let path_string = format!("/tokens/{}/fee", sell_token);
-        let post = || async {
-            request()
-                .path(&path_string)
-                .method("GET")
-                .reply(&filter)
-                .await
-        };
-        let response = post().await;
-        let body: handler::FeeInfo = serde_json::from_slice(response.body()).unwrap();
+    async fn get_fee_info_request_ok() {
+        let filter = get_fee_info_request();
+        let token = String::from("0x0000000000000000000000000000000000000000");
```

Comment:
nanonit: I found the test-case slightly nicer when it was not testing the Default value (which might be returned for other reasons).